pages/tic-tac-toe.tsx

```diff
@@ -0,0 +1,135 @@
+import { useRouter } from "next/router";
+import { useState } from "react";
+import styled from "styled-components";
+import { CenteredMain } from "../components/Centered";
+import { checkAnswer } from "../services/checkAnswer";
+
+const LOCK_LETTER = 'T'
+const Container = styled.div`
+    display: grid;
+    grid-template-columns: 200px 200px 200px;
+    grid-template-rows: 200px 200px 200px;
+`;
+
+const Cell = styled.div`
+    border: 1px solid black;
+    text-align: center;
+    vertical-align: middle;
+    font-size: 12em;
+    user-select: none;
+    cursor: pointer;
+`
+
+const XCell = styled(Cell)`color: blue`;
+const OCell = styled(Cell)`color: Purple`;
+
+const ClickedXCell = () => {
+    return <XCell>X</XCell>
+}
+
+const ClickedOCell = () => {
+    return <OCell>O</OCell>
+}
+
+const winningPatterns = [
```

Comment:
The patterns represent the indexes of the cells in 3x3 grid that need to be filled to win.